### PR TITLE
More instructions on what to open to see the project running added

### PR DESCRIPTION
**Pull request by mvktsk**
1. In Summury paragraph that sample only contains platform code, not storefront added.
2. New section "How to run Virto Commerce Platform Manager" with instructions how to browse 
platform +  screenshots of what you should expect to get added.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -70,9 +72,19 @@ Solution consists of 5 logically divided parts (projects):
 Developer should write code for a new module in .Core, .Data and .Web projects as usual in Visual Studio locally on host machine.
 Also developer should write tests for the new module in .Test project.
 
-## How to debug module
+## How to run Virto Commerce Platform Manager
+
+Docker bind Virto Commerce Platform Manager web app to 80 port. So if you have locally installed IIS server you should to stop it or set for local IIS other than 80 port.
```
Comment (eMazeika) [inline readme.md]: Docker would bind VC Platform Manager web app to port 80 by default. So, if you also have IIS server running locally, consider stopping it or changing the port in order to resolve port conflict.

@me: personally, I'm missing the option to change the port in docker. Could you add such instructions?

File: readme.md [hunk 2]
```diff
@@ -70,9 +72,19 @@ Solution consists of 5 logically divided parts (projects):
...
+
+Once images are started, open the Virto Commerce Platform Manager - http://localhost . This will launch the application with preinstalled default modules and give you opportunity to configure sample data.
```
Comment (eMazeika) [inline readme.md]: "Once images are started, open the Virto Commerce" -> "Once the containers are started, open VC"

File: readme.md (hunk 1 above)
Comment (mvktsk) [reply readme.md]: fixed

File: readme.md (hunk 2 above)
Comment (mvktsk) [reply readme.md]: fixed

**Pull request merged**